### PR TITLE
Feat: add BOB tokens to coreAssets.json

### PR DESCRIPTION
**Pull request by bvotteler**
Main change:
- added tokens in `bob` section of `coreAssets.json`

Additional changes:
- Updated existing references in `bob` section of `tokenMapper.js`
  - also changed coingecko id for SolvBTC.BBN from `solv-btc` to `solv-protocol-solvbtc-bbn`
- Updated BOB Gateway project's token list
  - mostly for future-proofing; should not result in any change to TVL

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/bob-gateway exports TVL: 
        
 
 
    bob                       2.27 M
    
    total                    2.27 M

**Comment by bvotteler** [comment]
@g1nt0ki , a question: Is there any harm or advantage if I added the info for the other tokens on BOB to `fixBalancesTokens`?

I'm not sure if that could negatively impact any other downstream code. 
I'd be happy to add their coingecko ids and decimals if it makes sense to do so.

File: projects/helper/tokenMapping.js [hunk 1]
```diff
@@ -79,9 +79,9 @@ const fixBalancesTokens = {
     '0xac485391eb2d7d88253a7f1ef18c37f4242d1a24': { coingeckoId: 'lisk', decimals: 18 },
   },
   bob: {
-    "0x541fd749419ca806a8bc7da8ac23d346f2df8b77": { coingeckoId: "solv-btc", decimals: 18 },
-    "0xcc0966d8418d412c599a6421b760a847eb169a8c": { coingeckoId: "solv-btc", decimals: 18 },
-    "0x236f8c0a61da474db21b693fb2ea7aab0c803894": { coingeckoId: "universal-btc", decimals: 8 },
+    [ADDRESSES.bob.SolvBTC]: { coingeckoId: "solv-btc", decimals: 18 },
+    [ADDRESSES.bob['SolvBTC.BBN']]: { coingeckoId: "solv-protocol-solvbtc-bbn", decimals: 18 },
```
Comment (g1nt0ki) [inline projects/helper/tokenMapping.js:83]: changing coingecko id is good, but prefer retaining address here

File: projects/bob-gateway/index.js [hunk 2]
```diff
@@ -31,10 +31,7 @@ module.exports = {
       return sumTokens2({
         api,
         chain: "bob",
-        tokens: [
-          ADDRESSES.bob.WBTC,
-          ADDRESSES.bob.TBTC,
-        ],
+        tokens: Object.values(ADDRESSES.bob),
```
Comment (g1nt0ki) [inline projects/bob-gateway/index.js:34]: hmm, this leads unexpected behavior if some key is removed from coreAssets file

**Comment by llamatester** [comment]
The adapter at projects/bob-gateway exports TVL: 
        
 
 
    bob                       2.27 M
    
    total                    2.27 M

File: projects/bob-gateway/index.js (hunk 2 above)
Comment (bvotteler) [reply projects/bob-gateway/index.js:34]: > hmm, this leads unexpected behavior if some key is removed from coreAssets file

Good point. I'll revert that change.

File: projects/helper/coreAssets.json [hunk 3]
```diff
@@ -1844,7 +1844,13 @@
     "STETH": "0x85008aE6198BC91aC0735CB5497CF125ddAAc528",
     "STONE": "0x96147a9ae9a42d7da551fd2322ca15b71032f342",
     "RETH": "0xb5686c4f60904ec2bda6277d6fe1f7caa8d1b41a",
-    "TBTC": "0xbba2ef945d523c4e2608c9e1214c2cc64d4fc2e2"
+    "TBTC": "0xbba2ef945d523c4e2608c9e1214c2cc64d4fc2e2",
+    "SolvBTC": "0x541fd749419ca806a8bc7da8ac23d346f2df8b77",
```
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:1848]: so, this file was never meant to be an address book, but feel like it is a losing battle atm, and I am throwing in the towel.

**Comment by g1nt0ki** [comment]
hi @bvotteler I should have mentioned that I was also working on the branch, included your changes here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/12294

**Comment by g1nt0ki** [comment]
> @g1nt0ki , a question: Is there any harm or advantage if I added the info for the other tokens on BOB to `fixBalancesTokens`?
> 
> I'm not sure if that could negatively impact any other downstream code. I'd be happy to add their coingecko ids and decimals if it makes sense to do so.

missed this comment, no negative impact.

**Pull request closed without merging**